### PR TITLE
feat(electron-update): call autoupdate after mainwindow is loaded

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

This PR fixes a small issue where `autoUpdate` was not getting called even once when the window was loaded and this would have created a situation where the autoupdate is just called after the user has spent an hour on the app, which is not what we want.

After this change, the check for update will be called once when the main window is loaded and that function also sets up an 1 hour recurring check

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Ran `yarn fastpass`
- [ ] Added/updated relevant unit test(s) (and ran `yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by smoralesd** [comment]
I saw another feat tag recently: (electron-update) is this the same feature using a different tag? Consistency on tagging PR's help whoever had to prepare release notes and the test plan.

**Comment by Shobhit1** [comment]
> I saw another feat tag recently: (electron-update) is this the same feature using a different tag? Consistency on tagging PR's help whoever had to prepare release notes and the test plan.

Updated

File: src/electron/main/main.ts [hunk 1]
```diff
@@ -41,6 +40,14 @@ const createWindow = () => {
...
         enableDevMode(mainWindow);
     });
+
+    electronAutoUpdateCheck
+        .check()
+        .then(() => {
+            console.log('checked for updates');
+            setupRecurringUpdateCheck();
+        })
+        .catch(console.log);
```
Comment (smoralesd) [inline src/electron/main/main.ts]: should this use `console.error`?

File: src/electron/main/main.ts [hunk 2]
```diff
@@ -41,6 +40,14 @@ const createWindow = () => {
         mainWindow.show();
         enableDevMode(mainWindow);
     });
+
+    electronAutoUpdateCheck
+        .check()
+        .then(() => {
+            console.log('checked for updates');
```
Comment (smoralesd) [inline src/electron/main/main.ts]: is this `console.log` needed?

Comment (Shobhit1) [reply src/electron/main/main.ts]: This wont be logged anywhere on the console, so doesn't hurt and its easier to check while developing! So, yes I am inclined to keep this

File: src/electron/main/main.ts (hunk 1 above)
Comment (Shobhit1) [reply src/electron/main/main.ts]: Thats how it was for the `loadUrl` catch, so following same patterns!

**Pull request merged**